src/Context/themeContext.js

```diff
@@ -1,24 +1,66 @@
-import { createContext, useState } from "react";
+// import { createContext, useState } from "react";
 
-export const ThemeContext = createContext()
+// export const ThemeContext = createContext()
 
-//Theme modes 
-const light = {mode: 'light', color: 'black', bg: 'white'}
-const dark = {mode: 'dark', color: 'white', bg: '#020300'}
+// //Theme modes 
+// const light = {mode: 'light', color: 'black', bg: 'white'}
+// const dark = {mode: 'dark', color: 'white', bg: '#020300'}
 
-//Get the system theme
-let systemTheme = {}
-window.matchMedia('(prefers-color-scheme: dark)').matches ? systemTheme = dark : systemTheme = light
+// //Get the system theme
+// let systemTheme = {}
+// window.matchMedia('(prefers-color-scheme: dark)').matches ? systemTheme = dark : systemTheme = light
 
+// export function ThemeProvider(props) {
```

Comment:
Don't leave comments if they are not adding any value to the code

Reply:
Ok I will remove it